Resources/views/macros/form_elements.html.twig

```diff
@@ -0,0 +1,12 @@
+{% macro recaptcha(siteKey)%}
+    <style type="text/css">
```

Comment:
We shouldn't have `<style>` outside the `<head>`.

Reply:
No problem with two forms on the same page, it works. Ok i will check this Bundle